src/include/security/levels.rb

```diff
@@ -65,127 +68,11 @@ def initialize_security_levels(include_target)
-            "net.ipv6.conf.all.forwarding"              => "0"
-          },
-          "Level3" => {
-            "CONSOLE_SHUTDOWN"                          => "ignore",
-            "CRACKLIB_DICT_PATH"                        => "/usr/lib/cracklib_dict",
-            "DISPLAYMANAGER_REMOTE_ACCESS"              => "no",
-            "kernel.sysrq"                              => "0",
-            "FAIL_DELAY"                                => "3",
-            "GID_MAX"                                   => "60000",
-            "GID_MIN"                                   => "1000",
-            "AllowShutdown"                             => "Root",
-            "HIBERNATE_SYSTEM"                          => "active_console",
-            "PASSWD_ENCRYPTION"                         => "sha512",
-            "PASSWD_USE_CRACKLIB"                       => "yes",
-            "PASS_MAX_DAYS"                             => "99999",
-            "PASS_MIN_DAYS"                             => "1",
-            "PASS_MIN_LEN"                              => "6",
-            "PASS_WARN_AGE"                             => "14",
-            "PERMISSION_SECURITY"                       => "secure",
-            "RUN_UPDATEDB_AS"                           => "nobody",
-            "UID_MAX"                                   => "60000",
-            "UID_MIN"                                   => "1000",
-            "SYS_UID_MAX"                               => "499",
-            "SYS_UID_MIN"                               => "100",
-            "SYS_GID_MAX"                               => "499",
-            "SYS_GID_MIN"                               => "100",
-            "USERADD_CMD"                               => "/usr/sbin/useradd.local",
-            "USERDEL_PRECMD"                            => "/usr/sbin/userdel-pre.local",
-            "USERDEL_POSTCMD"                           => "/usr/sbin/userdel-post.local",
-            "PASSWD_REMEMBER_HISTORY"                   => "0",
-            "SYSTOHC"                                   => "yes",
-            "SYSLOG_ON_NO_ERROR"                        => "yes",
-            "DISPLAYMANAGER_ROOT_LOGIN_REMOTE"          => "no",
-            "DISPLAYMANAGER_XSERVER_TCP_PORT_6000_OPEN" => "no",
-            "SMTPD_LISTEN_REMOTE"                       => "no",
-            "DISABLE_STOP_ON_REMOVAL"                   => "no",
-            "DISABLE_RESTART_ON_UPDATE"                 => "no",
-            "net.ipv4.tcp_syncookies"                   => "1",
-            "net.ipv4.ip_forward"                       => "0",
-            "net.ipv6.conf.all.forwarding"              => "0"
-          }
-        } 
+      @Levels = @LevelsNames.keys.each_with_object({}) do |level, levels|
+        lfile = Directory.find_data_file("security/#{level.downcase}.yml")
+        raise(Errno::ENOENT, "#{level.downcase}.yml file not found") unless lfile
```

Comment:
Is this extra error handling needed? YAML raises the exception itself:

```
irb(main):001:0> require "yaml"
=> true
irb(main):002:0> YAML.load_file "missingfile"
Errno::ENOENT: No such file or directory @ rb_sysopen - missingfile
```

Besides a very small difference in the exception message it's the same...